src/PgRvn.Server/RqlQuery.cs

```diff
@@ -91,128 +93,85 @@ private ICollection<PgColumn> GenerateSchema()
             if (_result.Results.Length == 0)
                 return Array.Empty<PgColumn>();
 
-
             var resultsFormat = GetDefaultResultsFormat();
-
             var sample = (BlittableJsonReaderObject)_result.Results[0];
 
-            if (_result.Includes.Count > 0 ||
-                sample.TryGet("@metadata", out BlittableJsonReaderObject metadata) && metadata.TryGet("@id", out string _))
+            if (sample.TryGet("@metadata", out BlittableJsonReaderObject metadata) && metadata.TryGet("@id", out string _) ||
```

Comment:
We can drop the support for includes, I think. That is no longer relevant for our use cases.

Reply:
Done